config/settings.yml

```diff
@@ -85,3 +85,7 @@ auth:
 # google_analytics_tracking_id: "someid"
 supplemental_files:
   proxy: false
+waveform:
+  player_width_in_px: 1200
+  finest_zoom_in_sec: 5
```

Comment:
Minor point, but since no other settings have unit in their names, should we shorten the names and add a comment, ie 
```
player_width: 1200 #pixels
finest_zoom: 5 #seconds
```